site/_posts/2017-04-07-blood-fire-manageiq.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Blood, fire and ManageIQ"
+author: panspagetka
+date: 2017-04-11
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+In the beginning there was nothing, which exploded. Same as this Last week in ManageIQ post.
+My name is [Robin](https://github.com/panspagetka), from the [UI team](https://github.com/ManageIQ/manageiq-ui-classic/). If you are reading this, I hope you are very enthusiastic and have very poor English grammar skill.
+
+I wanted to make an interview with the most famous person around here, George the Bunnygoat, but everything changed when fire nation attacked the weed fields. When I finally met him, he was high. He was so unbelievably high, that only thing I can get from him was , and I quote, this: [“Hey Dude.”.](https://www.youtube.com/watch?v=A_MjCqQoLLA)
```

Comment:
@cybette I think it's not that family unfriendly. You think that weed part is to much?